cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -138,9 +136,9 @@
   libreadline6-dev
 ]
 php_versions = %w[
-  5.4.45
-  5.5.30
-  5.6.24
+  5.6.31
+  7.0.21
+  7.1.7
```

Comment:
Strangely these versions aren't listed on:
http://php.net/releases/

However they are the listed (and the latest) on the homepage:
http://php.net/